lib/improver/metadata/constants/datatypes.py

```diff
@@ -28,25 +28,41 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""
-Module containing the default units and datatypes for use within IMPROVER.
+"""Module containing the default datatypes for use within IMPROVER."""
 
-The DEFAULT_UNITS dictionary has the following form.
+import numpy as np
 
-<str>:
-    The principle key is the name of a coordinate or diagnostic.
 
-"unit": <str>
-    The standard/default units for the coordinate or diagnostic
-    described by the key.  This is mandatory.
-"dtype": <dtype>
-    The standard/default data type in which the coordinate points
-    or diagnostic values should be stored.  This is optional; if
-    not set, float32 is assumed.
-"""
+# List of named quantities to be handled as int32 (all other numerical
+# fields are handled as float32)
+INTEGER_QUANTITIES = [
+    # present on external files
+    "realization",
+    "spot_index",
+    "weather_code",
+    # present on internal files / ancillaries only
+    "coefficient_index",
+    "grid_attributes",
+    "land_binary_mask",
+    "neighbour_selection_method",
+    "topography_mask"
+]
+
+
+TIME_METADATA = {
+    "time": {
+        "unit": "seconds since 1970-01-01 00:00:00",
+        "dtype": np.int64},
+    "forecast_reference_time": {
+        "unit": "seconds since 1970-01-01 00:00:00",
+        "dtype": np.int64},
+    "forecast_period": {
+        "unit": "seconds",
+        "dtype": np.int32}
+}
 
-import numpy as np
 
+# DEPRECATED
 DEFAULT_UNITS = {
```

Comment:
Yes I think we should remove this before merging the PR.  I kept it through review in case the information might want to be transferred / saved elsewhere.

Reply:
👍 OK, but let's not forget about removing it before merge.